File: JAVA/Motherboard.java
public class Motherboard extends Komponen {
    private String chipset;
    private String soket;

    public Motherboard() {}

    public Motherboard(String chipset, String soket, String merk, String nama) {
        super(merk, nama);
        this.chipset = chipset;
        this.soket = soket;
    }

    public void setChipset(String chipset) {
        this.chipset = chipset;
    }

    public void setSoket(String soket) {
        this.soket = soket;
    }

    public String getChipset() {
        return chipset;
    }

    public String getSoket() {
        return soket;
    }
}

File: PYTHON/Motherboard.py
from Komponen import Komponen

class Motherboard(Komponen):
    def __init__(self, chipset="", soket="", merk="", nama=""):
        super().__init__(merk, nama)
        self.chipset = chipset
        self.soket = soket

    def set_chipset(self, chipset):
        self.chipset = chipset

    def set_soket(self, soket):
        self.soket = soket

    def get_chipset(self):
        return self.chipset

    def get_soket(self):
        return self.soket
